aten/src/ATen/native/native_functions.yaml

```diff
@@ -685,6 +685,11 @@
     SparseCPU: empty_sparse
     SparseCUDA: empty_sparse
 
+- func: empty_affine_quantized(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, float scale=1, int zero_point=0) -> Tensor
+  matches_jit_signature: True
+  dispatch:
+    CPU: empty_affine_quantized_cpu
```

Comment:
@gchanan is the dispatch key here computed from the TensorOptions argument? we used CPU because we TensorOptions can't express QuantizedCPU, is that OK?

Reply:
I hacked around this for complex in #19275, but maybe we need a generalized solution.